yt_dlp/extractor/bigo.py

```diff
@@ -0,0 +1,56 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..utils import (
+    urlencode_postdata,
+    ExtractorError,
+)
+
+
+class BigoIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?bigo\.tv/(?:[a-z]{2,}/)?(?P<id>[^/]+)'
+    # Note: I would like to provide some real test, but Bigo is a live streaming
+    # site, and a test would require that the broadcaster is live at the moment.
+    # So, I don't have a good way to provide a real test here.
+    _TESTS = [{
+        'url': 'https://www.bigo.tv/th/Tarlerm1304',
+        'only_matching': True,
+    }, {
+        'url': 'https://bigo.tv/115976881',
+        'only_matching': True,
+    }]
```

Comment:
Live tests should generally still have fields set, but with `skip` set instead of `only_matching`
See https://github.com/yt-dlp/yt-dlp/pull/866/files#diff-851636b1dfe58a3ee8afee816fe4d26527b3c0515f6a47aada2854acf84077d6R12